packages/wallets/src/evm/wallets/metamask.ts

```diff
@@ -20,6 +20,12 @@ export class MetaMask extends AbstractBrowserWallet {
   isInjected?: boolean;
   walletConnectConnector?: WalletConnectV1ConnectorType;
 
+  static meta = {
+    name: "Metamask",
+    iconURL:
+      "ipfs://QmZZHcw7zcXursywnLDAyY6Hfxzqop5GKgwoq8NB9jjrkN/metamask.svg",
```

Comment:
I think we *may* want to look at just having these as local base64 blobs / buffers hard-coded in the longer term, that'll still work with img.src regadless though so that's probably fine for now